_layouts/software-details.html

```diff
@@ -31,7 +31,7 @@
         {% else %}
         {% assign logo = page.publiccode.logo %}
         {% endif %}
-        <img src="{{ page.publiccode.logo }}" class="d-inline-block" />
+        <img src="{{ logo }}" class="d-inline-block" />
```

Comment:
Shoudn't this use `display-logo.html`?

Reply:
atm `display-logo.html` have a different format and it can be used only for listing